core/src/main/java/bisq/core/network/p2p/inventory/messages/GetInventoryResponse.java

```diff
@@ -28,9 +28,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import lombok.EqualsAndHashCode;
 import lombok.Value;
 
 @Value
```

Comment:
That should also be replaced by a @Getter and @ToString.

Reply:
Same as above...  commit  c2c1ac2 
Removes `@Value`, `@EqualsAndHashCode(callSuper = false)`
Adds `@Getter`,`@ToString`

I'm not absolutely certain this is what you asked for (by removing `@Value`), but keeping `@Value` , `@Getter`, `@ToString` is redundant because `@Getter`, `@ToString` are implicitly defined by `@Value`.

Reply:
Ah sorry I see now the confusion as the marked line was EqualsAndHashCode instead of Value. My fault... sorry...